packages/modal/src/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta } from "@storybook/react";
+import { Button } from "@telegraph/button";
+import { Heading } from "@telegraph/typography";
+
+import { Modal as TelegraphModal } from "./Modal";
+
+const meta: Meta<typeof TelegraphModal> = {
+  title: "Components/Modal",
+  // component: TelegraphModal,
+};
+export default meta;
+
+export const Modal = {
+  render: () => (
+    <TelegraphModal.Root open={true}>
```

Comment:
Nit: Would be nice to have an open/close button to fully test out the modal and show the open/close animation.